docs/src/browsers.md

```diff
@@ -22,7 +22,47 @@ on when to opt into stable channels, refer to the [Google Chrome & Microsoft Edg
 ## Firefox
 
 Playwright's Firefox version matches the recent [Firefox Beta](https://www.mozilla.org/en-US/firefox/channel/desktop/)
-build.
+build. 
+
+### Firefox-Stable
+
+Playwright team maintains a Playwright Firefox version that matches the latest Firefox Stable, a.k.a. `firefox-stable`.
+
+Using `firefox-stable` is a 2-steps process:
+
+1. Installing `firefox-stable` with Playwright CLI.
+    ```shell
```

Comment:
It is possible to add language-specific shell scripts, see here in the bottom of the file: https://github.com/microsoft/playwright/blob/7405655c26994a418dbeb93ad2a6cb87c77587cf/docs/src/cli.md

Reply:
done!